Traits/MinkTrait.php

```diff
@@ -94,7 +94,10 @@ public function isElementVisible($element)
         //select with css
         $docSelector = "document.querySelectorAll('".$element."')";
         $query = $docSelector.".length > 0 && ".$docSelector."[0].offsetWidth !== 0";
-        $session->wait($this->timeout, $query);
+
+        if (!$session->wait($this->timeout, $query)) {
+            throw $this->elementNotFound('element', 'css', $element);
```

Comment:
The element not found is not correct, it should an element not visible exception. The getElementByCss should be done before to assert the fact that the element exists in DOM